### PR TITLE
Clean up temp directories.

### PR DESCRIPTION
**Pull request by ddunkin**

### CODE REVIEW

File: src/Faithlife.PackageDiffTool/LocalPackageHelper.cs [hunk 1]
```diff
@@ -96,9 +99,24 @@ public async Task<PackageReaderBase> GetLocalPackageAsync(string packagePath, Ca
...
 
+		public void Dispose()
+		{
+			foreach (var directory in m_tempDirectories)
+			{
+				try
+				{
+					Directory.Delete(directory, true);
+				}
+				catch (IOException)
```
Comment (bgrainger) [inline src/Faithlife.PackageDiffTool/LocalPackageHelper.cs]: FWIW we catch more exception types here: https://github.com/Faithlife/FaithlifeUtility/blob/eadb97bb7c2076ce61e4d8e4f966133479706564/src/Faithlife.Utility/DirectoryUtility.cs#L24-L34

File: src/Faithlife.PackageDiffTool/Program.cs [hunk 2]
```diff
@@ -21,77 +21,78 @@ public static Task<int> Main(string[] args)
 
 		static async Task<int> RunAsync(Options options)
 		{
-			var packageHelper = new LocalPackageHelper
+			using (var packageHelper = new LocalPackageHelper())
```
Comment (bgrainger) [inline src/Faithlife.PackageDiffTool/Program.cs]: `using var packageHelper = new LocalPackageHelper();` :)

Comment (bgrainger) [reply src/Faithlife.PackageDiffTool/Program.cs]: (Would using C# 8 here depend on the other PR that switches to .NET Core only?)

Comment (ddunkin) [reply src/Faithlife.PackageDiffTool/Program.cs]: I wrote it that way initially, but the CI build didn't support C# 8. With #14 merged, it does.

**Pull request merged**